src/components/settings/SettingsAppearanceTab/index.jsx

```diff
@@ -1,48 +1,121 @@
+  const curatedThemes = [ACUMEN_THEME, CAMBRIDGE_THEME, IMPACT_THEME, PIONEER_THEME, SAGE_THEME, SCHOLAR_THEME];
+
+  function getStartingTheme() {
+    for (let i = 0; i < curatedThemes.length; i++) {
+      if (curatedThemes[i].button === enterpriseBranding.primary_color
+        && curatedThemes[i].banner === enterpriseBranding.secondary_color
+        && curatedThemes[i].accent === enterpriseBranding.tertiary_color) {
+        return [curatedThemes[i], null];
+      }
+    }
+    const CUSTOM_THEME = {
+      title: 'Custom Theme',
+      button: enterpriseBranding.primary_color,
+      banner: enterpriseBranding.secondary_color,
+      accent: enterpriseBranding.tertiary_color,
+    };
+    return [CUSTOM_THEME, CUSTOM_THEME];
+  }
 
-  async function handleProcessUpload({
+  // this variable is [selected , null] if the user has not created a custom
+  // theme and it is [selectedTheme, [customName, #primary, #secondary, #tertiary]] if they have
+  // (they can't be two variables because of rules regarding setting react hooks)
+  const [theme, setTheme] = useState(getStartingTheme());
+
+  async function handleLogoUpload({
     fileData, handleError,
   }) {
     try {
-      const response = await LmsApiService.updateEnterpriseCustomerBranding(enterpriseId, fileData);
+      const formData = new FormData();
+      formData.append('logo', fileData.get('file'));
+      const response = await LmsApiService.updateEnterpriseCustomerBranding(enterpriseId, formData);
       if (response.status === 204) {
+        setUploadedFile(fileData.get('file'));
         setConfigChangeSuccess(true);
       }
     } catch (error) {
       handleError(error);
     }
   }
 
+  const saveChanges = () => {
+    const sendThemeData = async (formData) => {
+      const response = await LmsApiService.updateEnterpriseCustomerBranding(enterpriseId, formData);
+      if (response.status === 204) {
+        const updatedBranding = {
+          logo: uploadedFile || enterpriseBranding.logo,
+          primary_color: theme[0].button,
+          secondary_color: theme[0].banner,
+          tertiary_color: theme[0].accent,
+        };
+        updatePortalConfiguration({ enterpriseBranding: updatedBranding });
+        setConfigChangeSuccess(true);
+      } else {
+        setConfigChangeSuccess(false);
+      }
+    };
+    try {
+      const formData = new FormData();
+      formData.append('primary_color', theme[0].button);
+      formData.append('secondary_color', theme[0].banner);
+      formData.append('tertiary_color', theme[0].accent);
+      sendThemeData(formData);
+    } catch {
+      setConfigChangeSuccess(false);
+    }
+  };
+
   return (
     <>
       <h2 className="py-2">Portal Appearance</h2>
       <p>
         Customize the appearance of your learner and administrator edX experiences with your
         organization’s logo and color themes.
       </p>
+      {configChangeSuccess === false && (
+      <Alert
```

Comment:
Let's use the `show` prop to control the `Alert` visibility rather than conditionally rendering the entire `Alert` component. It turns out this approach (versus using `show={true/false}` is that we lose the animation fade in/out.